styles/styles.css

```diff
@@ -332,3 +332,17 @@ main .section {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+div.experts-page-heading {
+  background: #f8f8f8;
+  letter-spacing: 2px;
+  padding-bottom: 13px;
+  padding-top: 50px;
+}
+
+div.experts-page-editorial-staff {
+  margin: 0;
+  padding: 0;
+  margin-top: -5px;
```

Comment:
It’s an anti-pattern to have negative values. Maybe we can omit it, or remove space somewhere else. I need to check next week